### ISSUE TITLE
All 404 paths go to a page requesting an issue submission for a broken link

### ISSUE THREAD

**Issue by SeanGallen** [graphql/graphql.github.io#1753]
I notice a few issues suggesting a broken link but I could not find these links.  After exploring the site, I notice any 404 path in the URL shows a Page Not Found UI **with** the request to submit an issue for a broken link. On the 404 page there is a link which creates a boilerplate issue.  
Example: https://graphql.org/random-path/ generates an issue titled  `Found broken /random-path link. Please fix!`

I cannot speak directly to issues https://github.com/graphql/graphql.github.io/issues/1726 and https://github.com/graphql/graphql.github.io/issues/1748, but maybe the wording should change or the page should redirect to the home page? People may have bookmarked pages which don't exist anymore but the site suggests something is broken. This could create a lot of unnecessary issues.
Either way, I could be missing something so I wanted to raise the question. Thanks for all the hard work.

**Comment by benjie** [comment]
We should never break links; if a page once existed and no longer does, we should redirect to the newest resource that replaces it. I've no idea where the `/debugging` link came from (`git log -S'/debugging'` did not turn up any hits) or if that page ever really existed, but to help us catch these issues I raised:

- https://github.com/graphql/graphql.github.io/pull/1810

to include the referring page. Mostly we want to be notified if we've accidentally deleted a page that someone had bookmarked.

**Comment by saihaj** [comment]
Agreed with @benjie in an ideal world we wouldn't break any links. I added a task so in some framework migration we never break a page https://github.com/graphql/graphql.github.io/issues/1911 this will help us ensure in CI that we don't break anything before merging. 

As for Issue templates in a worst case or for any old stuff where folks have pages bookmarked and we broke the page we would want to know how it happened so we do want to keep that flow.
